### DISCUSSION TITLE
2024/09/01/20240901_updatelog/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#26] [General]
# 2024/09/01/20240901_updatelog/

八月份过得好漫长啊。 最近倒是写下了不少碎碎念，浅浅整理于此，就当是对这个炎热的夏天的一些回忆吧。

https://wz.anoms.top/2024/09/01/20240901_updatelog/

**Comment by Hazelandinstant** [comment]
actually it's called "课题分离" :)

**Reply by cyclinbox** [reply]
> 课题分离：在处理人际关系时，明确划分和尊重个人之间的课题边界。这要求个体区分自己的课题（即自己需要负责和解决的事情）和他人的课题（即他人需要负责和解决的事情
>
> 客体分离：个体在成长过程中，逐渐认识到自己与母亲（或其他主要抚养者）是两个独立的个体，并发展出独立自我意识和个体性的过程。它涉及到个体在心理发展过程中逐渐区分自我与他人的过程。

还真的是，我给记混了。谢谢指正！ :)

（AND转博加油！）
